internal/metamorph/processor.go

```diff
@@ -646,9 +669,9 @@ func (p *Processor) StartProcessExpiredTransactions() {
 	}()
 }
 
-// GetPeers returns a list of connected and a list of disconnected peers
-func (p *Processor) GetPeers() []p2p.PeerI {
-	return p.pm.GetPeers()
+// GetPeersNmber returns a list of connected and a list of disconnected peers
+func (p *Processor) GetPeersNmber() uint {
+	return p.pm.CountConnectedPeers()
 }
```

Comment:
I don't know if it was on purpose, but just in case 😄 
```suggestion
// GetPeersNumber returns a list of connected and a list of disconnected peers
func (p *Processor) GetPeersNumber() uint {
	return p.pm.CountConnectedPeers()
}
```